common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_action.h

```diff
@@ -50,7 +50,6 @@ enum eActionOp_CONTROL: uint8_t {
     ACTION_CONTROL_CHANGE_MODULE_LOGGING_LEVEL = 0xb,
     ACTION_CONTROL_ARP_QUERY_REQUEST = 0xc,
     ACTION_CONTROL_ARP_QUERY_RESPONSE = 0xd,
-    ACTION_CONTROL_PLATFORM_OPERATIONAL_NOTIFICATION = 0xe,
     ACTION_CONTROL_BACKHAUL_ROAM_REQUEST = 0x1e,
     ACTION_CONTROL_BACKHAUL_DL_RSSI_REPORT_NOTIFICATION = 0x1f,
```

Comment:
this was raised (and handled) in other PRs - please merge-together the tlvf changes and the Auto-generated files